HunterJuneau/dine-and-dash
# Added private routes and admin only views

## Description

Restricted admin views for admins only and added private route functionality.

## Related Issue

- #86 

## Motivation and Context

Prevents regular users from accessing the admin views

## How Can This Be Tested?

Add the following items to your `.env` file: 
```
REACT_APP_ADMIN_UID_BEN=
REACT_APP_ADMIN_UID_HUNTER=
REACT_APP_ADMIN_UID_CHRIS=
```
Grab the UID's from the Firebase and fill them in, then you should be good to go.

## Types of changes
